de-lib/src/main/java/org/iplantc/de/tools/client/views/AppUsedByAToolView.java

```diff
@@ -0,0 +1,16 @@
+package org.iplantc.de.tools.client.views;
```

Comment:
Was this class created for some later changes or is there something missing here?